.github/workflows/slowtest.yaml

```diff
@@ -22,7 +22,7 @@ env:
   RUST_LOG: info,libp2p=off,node=error
 
 jobs:
-  slow-tests:
+  slowest:
```

Comment:
I think some unwanted change may have been pulled in here.

Reply:
Seems likely, I can revert this